CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 Bugfixes:
 - Make ProxyUtil.IsAccessible(MethodBase) take into account declaring type's accessibility so it doesn't report false negatives for e.g. public methods in inaccessible classes. (@stakx, #289)
+- Fix System.InvalidCastException during Invocation.ChangeProxyTarget method call (@yallie, #293)
```

Comment:
Probably best to write include `IChangeProxyTarget` in parenthesis (or something) since it isn't obvious this is how you access it because it isn't part of `IInvocation`.